types/time_test.go

```diff
@@ -1101,11 +1101,59 @@ func (s *testTimeSuite) TestCheckTimestamp(c *C) {
 	for _, t := range tests {
 		validTimestamp := types.CheckTimestampTypeForTest(&stmtctx.StatementContext{TimeZone: t.tz}, t.input)
 		if t.expectRetError {
-			c.Assert(validTimestamp, NotNil, Commentf("For %s", t.input))
+			c.Assert(validTimestamp, NotNil, Commentf("For %s %s", t.input, t.tz))
 		} else {
-			c.Assert(validTimestamp, IsNil, Commentf("For %s", t.input))
+			c.Assert(validTimestamp, IsNil, Commentf("For %s %s", t.input, t.tz))
 		}
 	}
+
+	// Issue #13605: "Invalid time format" caused by time zone issue
+	// Some regions like Los Angeles use daylight saving time, see https://en.wikipedia.org/wiki/Daylight_saving_time
+	losAngelesTz, _ := time.LoadLocation("America/Los_Angeles")
+	rawLocal := time.Local
+	time.Local = losAngelesTz // Hack Local timezone
+
+	tests = []struct {
```

Comment:
How about adding a new test for London DST?

> March 31, 2019, 1:00:00 am clocks were turned forward 1 hour to March 31, 2019, 2:00:00 am local daylight time instead.

Reply:
@bb7133 comment addressed, PTAL